route-rs-runtime/src/element/identity_elements.rs

```diff
@@ -1,26 +1,29 @@
 use crate::element::{AsyncElement, Element};
+use std::fmt::Debug;
 use std::marker::PhantomData;
+use tracing::instrument;
 
 /* IdentityElement
   This is an element that passes what it has received
 */
 #[derive(Default)]
-pub struct IdentityElement<A: Sized> {
+pub struct IdentityElement<A: Sized + Debug> {
```

Comment:
This is here because the `#[instrument]` macro treats the arguments to the function it's instrumenting as `span` metadata. Tose values are required to be `Debug`.

Reply:
That's fine, we will probably use debug a lot anyways. It is a useful thing.